### PR TITLE
Remove unused param warning

### PR DESCRIPTION
**Pull request by tayloryoung6396**

### CODE REVIEW

File: shared/utility/motion/InverseKinematics.cpp [hunk 1]
```diff
@@ -337,7 +337,7 @@ namespace utility::motion::kinematics {
         joints.insert(joints.end(), joints2.begin(), joints2.end());
         return joints;
     }
-    std::vector<std::pair<ServoID, double>> calculateCameraLookJoints(const KinematicsModel& model,
+    std::vector<std::pair<ServoID, double>> calculateCameraLookJoints(const KinematicsModel&,  // model,
```
Comment (KipHamiltons) [inline shared/utility/motion/InverseKinematics.cpp:340]:
```suggestion
    std::vector<std::pair<ServoID, double>> calculateCameraLookJoints(const KinematicsModel&  /*model*/,
```

**Review by KipHamiltons — COMMENTED** [review]
Last time I looked at removing this warning, IIRC @Bidski said that this function really should be using this parameter, and thus the warning was a valid reminder that it should be dealt with. :shrug: 
EDIT: I did not remember correctly :sweat_smile: see his comment

**Comment by Bidski** [comment]
> Last time I looked at removing this warning, IIRC @Bidski said that this function really should be using this parameter, and thus the warning was a valid reminder that it should be dealt with. :shrug: 

What i said was that it probably was used at one point, but obviously its not needed any more.

Rather than commenting it, you should just remove it

**Comment by ysims** [comment]
> > Last time I looked at removing this warning, IIRC @Bidski said that this function really should be using this parameter, and thus the warning was a valid reminder that it should be dealt with. 🤷
> 
> What i said was that it probably was used at one point, but obviously its not needed any more.
> 
> Rather than commenting it, you should just remove it

Sorry team, I created this warning last year when finishing off the camera calibration PR Trent did. IK used to do the whole thing to the cameras and took the last transform from the kinematics model, but now vision does the last part from the neck pitch servo to the cameras and takes the transform from the Hcp matrix in the camera config. This might have affected head behaviour though?

**Comment by KipHamiltons** [comment]
why did you close this @tayloryoung6396 ? why not just remove the parameter

**Comment by tayloryoung6396** [comment]
Because 

> > > Last time I looked at removing this warning, IIRC @Bidski said that this function really should be using this parameter, and thus the warning was a valid reminder that it should be dealt with. shrug
> > 
> > 
> > What i said was that it probably was used at one point, but obviously its not needed any more.
> > Rather than commenting it, you should just remove it
> 
> Sorry team, I created this warning last year when finishing off the camera calibration PR Trent did. Kinematics used to do the whole thing to the cameras and took the last transform from the kinematics model, but now vision does the last part from the neck pitch servo to the cameras and takes the transform from the Hcp matrix in the camera config. This might have affected head behaviour though?

**Comment by Bidski** [comment]
Still needs fixing?

**Review by tayloryoung6396 — COMMENTED** [review]
LGTM

**Review by KipHamiltons — APPROVED** [review]

**Pull request merged**